src/localization/fi.json

```diff
@@ -167,7 +167,8 @@
 
   "exhibition": {
     "reception": "Vastaanotto",
-    "visitorVariablesManagement": "Vierailijan valintojen hallinta",
+    "visitorVariables": "Vierailijan valinnat",
```

Comment:
Visitor variables and visitor varialbe management sound quite similar, should this be called, e.g. Muuttujien hallinta?